src/command/render/latexmk/texlive.ts

```diff
@@ -10,6 +10,7 @@ import * as ld from "../../../core/lodash.ts";
 import { execProcess } from "../../../core/process.ts";
 import { kLatexHeaderMessageOptions } from "./types.ts";
 import { lines } from "../../../core/text.ts";
+import { removeIfExists } from "../../../core/path.ts";
 
 const tlmgr = Deno.build.os === "windows"
```

Comment:
I did not touch this function but it is now only used by the following `hasTexLive()` which can probably be rewritten with 

````ts
await tlmgrCommand({
  cmd: ["--version"]
})
````

but it was not part of the issue so I did not touched.

Not sure about the policy about trying to leave as much as possible for tiny changes, or trying to refactor a bit on the go when we modify code base like that. You'll tell me.

Reply:
ok I let it as is. thanks

Reply:
Done